src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala

```diff
@@ -16,7 +16,7 @@ class EvenFibonnacciTest extends WordSpec {
 
     "return the right value for n = 10" in {
       assert(
```

Comment:
🙅 List(1, 1, 2, 3, 5, 8, 13, 21, 34, 55) did not equal List(1, 1, 2, 3, 4, 8, 13, 21, 34, 55)